lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -1038,6 +1039,7 @@ withSystemTempDir tr name action = do
 data ClusterLog
     = MsgCartouche
     | MsgLauncher String LauncherLog
+    | MsgServerForMetadataStarted FilePath
```

Comment:
Might be good to log both the directory and url - the url type being String.

Reply:
👍 now
```
cardano-wallet.integration:Info:43] [2020-06-23 09:01:53.84 UTC] Started a static server for http://localhost:60431/ at /private/var/folders/h_/mlzym27d5lz_fx3ph2d547700000gn/T/integration-62aa8274a6e3655b/stake-pool-2
[cardano-wallet.integration:Info:41] [2020-06-23 09:01:53.84 UTC] Started a static server for http://localhost:60430/ at /private/var/folders/h_/mlzym27d5lz_fx3ph2d547700000gn/T/integration-62aa8274a6e3655b/stake-pool-0
[cardano-wallet.integration:Info:46] [2020-06-23 09:01:53.84 UTC] Started a static server for http://localhost:60432/ at /private/var/folders/h_/mlzym27d5lz_fx3ph2d547700000gn/T/integration-62aa8274a6e3655b/stake-pool-1
```